frontend/src/scenes/insights/Trends.cy-spec.js

```diff
@@ -0,0 +1,353 @@
+                            type: 'events',
+                            order: 0,
+                        },
+                    ]),
+                    properties: JSON.stringify([
+                        {
+                            key: '$browser',
+                            value: 'Chrome',
+                            operator: 'exact',
+                            type: 'event',
+                        },
+                    ]),
+                })
+
+            cy.get('[data-attr="trend-line-graph"]').should('be.visible')
+            cy.contains('Chrome').should('be.visible')
+        })
+
+        it('renders multiple prop', () => {
+            helpers.setLocation('/insights', {
+                insight: 'TRENDS',
+                interval: 'day',
+                display: 'ActionsLineGraph',
+                events: [
+                    {
+                        id: '$pageview',
+                        name: '$pageview',
+                        type: 'events',
+                        order: 0,
+                    },
+                ],
+                properties: [
+                    {
+                        key: '$browser',
+                        value: 'Chrome',
+                        operator: 'exact',
+                        type: 'event',
+                    },
+                    {
+                        key: '$current_url',
+                        value: 'http://posthog.com',
+                        operator: 'exact',
+                        type: 'event',
+                    },
+                ],
+            })
+            mount()
+
+            cy.wait('@api_insight')
+                .map(helpers.getSearchParameters)
+                .should('include', {
+                    insight: 'TRENDS',
+                    interval: 'day',
+                    display: 'ActionsLineGraph',
+                    events: JSON.stringify([
+                        {
+                            id: '$pageview',
+                            name: '$pageview',
+                            type: 'events',
+                            order: 0,
+                        },
+                    ]),
+                    properties: JSON.stringify([
+                        {
+                            key: '$browser',
+                            value: 'Chrome',
+                            operator: 'exact',
+                            type: 'event',
+                        },
+                        {
+                            key: '$current_url',
+                            value: 'http://posthog.com',
+                            operator: 'exact',
+                            type: 'event',
+                        },
+                    ]),
+                })
+
+            cy.get('[data-attr="trend-line-graph"]').should('be.visible')
+            cy.contains('http://posthog.com').should('be.visible')
+        })
+
+        it('responsive to shown as', () => {
```

Comment:
Naming suggestion: Test names together with `describe` should become full sentences which make sense

Current:
> <Insights /> trends Trend filters from url responsive to shown as

Recommendation:
> <Insights /> trends page filter in url responds to shown_as parameter

So rename the describe to `filter in url` and it to `responds to shown_as parameter`